src/content/docs/tips/tailwind-v4.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Tailwind v4
+description: Quick set up guide for Tailwind v4
+astroRange: ^4.0.0
+---
+import { Tabs, TabItem, Steps } from "@astrojs/starlight/components";
+
+How to use Tailwind's v4 Vite plugin instead of Astro's Tailwind integration.
```

Comment:
I think it's worth explaining quickly? Like it makes the integration basically useless now